org.eclipse.jgit.pgm/src/org/eclipse/jgit/pgm/Merge.java

```diff
@@ -72,6 +82,26 @@ void ffonly(@SuppressWarnings("unused") final boolean ignored) {
 	/** {@inheritDoc} */
 	@Override
 	protected void run() {
+		if (abort) {
+			try {
+				if (abortActiveMerge()) {
+					System.out.println(CLIText.get().mergeAborted);
+					return;
+				}
+			} catch (Exception e) {
+				System.out.println(e.getMessage());
+				System.out.println(CLIText.get().useHelpCommand);
+			}
+			return;
+		}
+
+		// If ref is empty, print message and escape
+		if (ref == null) {
+			System.out.println(CLIText.get().mergeSpecifyBranch);
+			System.out.println(CLIText.get().useHelpCommand);
```

Comment:
small nit. I would suggest storing `CLIText.get()`, seems like it's used a lot.

Reply:
I agree, but CLIText is a JGit method used to get strings from a separate file. This is to make it easier for JGit to be translated to other languages. Since CLIText is an established method in JGit, I did not think it was worth modifying.